pkgs/development/python-modules/berkeleydb/default.nix

```diff
@@ -21,8 +22,6 @@ buildPythonPackage rec {
     setuptools
   ];
 
-  buildInputs = [ pkgs.db ];
```

Comment:
That commit accidentally removed the buildInput

Reply:
It was intended to be removed as @TomaSajt did recommend previously, please read the history of this PR.